src/ui/connectionmanager.cc

```diff
@@ -184,12 +184,14 @@ void ConnectionManager::startLocalServer() {
   if (!check_file.exists()) {
     python_interpreter_executable = QString("py.exe");
   }
-#else
+#elif defined(Q_OS_MAC)
   QString python_interpreter_executable(qApp->applicationDirPath() + "/../Resources/veles-server/venv/bin/python3");
   QFileInfo check_file(python_interpreter_executable);
   if (!check_file.exists()) {
     python_interpreter_executable = QString("python3");
   }
+#else
+  #error OS not supported
```

Comment:
IMO it's still better to fail explicitly here, so user will have to fix it for strange OSes. Also, in case we port Veles to something other we won't forget to fix this place.